packages/core/chrome/core-chrome-browser-internal/src/ui/project/navigation.tsx

```diff
@@ -6,67 +6,52 @@
  * Side Public License, v 1.
  */
 
-import React, { useCallback } from 'react';
-import useLocalStorage from 'react-use/lib/useLocalStorage';
+import React from 'react';
 import { css } from '@emotion/react';
-
-import { i18n } from '@kbn/i18n';
-import { EuiButtonIcon, EuiCollapsibleNav } from '@elastic/eui';
-
-const LOCAL_STORAGE_IS_OPEN_KEY = 'PROJECT_NAVIGATION_OPEN' as const;
-const SIZE_OPEN = 248;
-const SIZE_CLOSED = 40;
-
-const buttonCSS = css`
-  margin-left: -32px;
-  margin-top: 12px;
-  position: fixed;
-  z-index: 1000;
-`;
-
-const openAriaLabel = i18n.translate('core.ui.chrome.projectNav.collapsibleNavOpenAriaLabel', {
-  defaultMessage: 'Close navigation',
-});
-
-const closedAriaLabel = i18n.translate('core.ui.chrome.projectNav.collapsibleNavClosedAriaLabel', {
-  defaultMessage: 'Open navigation',
-});
-
-export const ProjectNavigation: React.FC = ({ children }) => {
-  const [isOpen, setIsOpen] = useLocalStorage(LOCAL_STORAGE_IS_OPEN_KEY, true);
-
-  const toggleOpen = useCallback(() => {
-    setIsOpen(!isOpen);
-  }, [isOpen, setIsOpen]);
-
+import { EuiCollapsibleNav, EuiCollapsibleNavProps, useIsWithinMinBreakpoint } from '@elastic/eui';
+
+const SIZE_EXPANDED = 248;
+const SIZE_COLLAPSED = 48;
+
+export interface ProjectNavigationProps {
+  isOpen: boolean;
+  closeNav: () => void;
+  button: EuiCollapsibleNavProps['button'];
+}
+
+export const ProjectNavigation: React.FC<ProjectNavigationProps> = ({
+  children,
+  isOpen,
+  closeNav,
+  button,
+}) => {
   const collabsibleNavCSS = css`
     border-inline-end-width: 1,
     display: flex,
     flex-direction: row,
   `;
 
+  // on small screen isOpen hides the nav,
+  // on larger screen isOpen makes it smaller
```

Comment:
minor nit: It seems like these 2 comments should be moved down by 2 lines for closer connection to where `isOpen` becomes important.